database/migrations/2018_05_16_153409_add_first_counter_totals_to_assets.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use App\Models\Asset;
+
+class AddFirstCounterTotalsToAssets extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // This artisan call may take a while
```

Comment:
Is it possible to put some console output here from the migration warning users about the delay? Not sure..

Reply:
Things don't always output as expected in migrations. Could try tho.